packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -111,6 +117,24 @@ class AttestationRequestHandler {
     // TODO: Check expiration
   }
 
+  private async verifyPepperIfApplicable(): Promise<void> {
+    if (this.attestationRequest.blindedPhoneNumberSignature) {
+      try {
+        const phoneNumberHashDetails = await getPhoneNumberIdentifierFromSignature(
+          this.attestationRequest.phoneNumber,
+          this.attestationRequest.blindedPhoneNumberSignature,
+          blsBlindingClient
+        )
+        if (phoneNumberHashDetails.pepper !== this.attestationRequest.salt) {
+          throw new ErrorWithResponse('Pepper is invalid', 422)
```

Comment:
Let's silently log this for now as an error in the logs, just to make sure we don't accidentally block users until we're confident. Can you add a ticket to remind us to check up on this once it's released?